lib/models/application.js

```diff
@@ -1562,6 +1565,141 @@ const getApplicationModel = function (deps, opts) {
 			 */
 			remove: envVarModel.remove,
 		},
+
+		/**
+		 * @namespace balena.models.application.invite
+		 * @memberof balena.models.application
+		 */
+		invite: {
```

Comment:
@iamsolankiamit You probably need to check https://github.com/balena-io/balena-sdk/pull/901 which adds back callback support for nested namespaces (essentially just wrap with an `addCallbackSupportToModule` call).